sdk/download.go

```diff
@@ -318,3 +319,18 @@ func UpdateSignedFile(file, url string, client *http.Client) error {
 	plog.Infof("Verified file: %s", file)
 	return nil
 }
+
+func DetectSDKVersion(url, branch string) (string, error) {
```

Comment:
It may be behind but the subsequent update_chroot and/or build_packages will upgrade the chroot to (hopefully) the same state either way. The update_chroot command always happens as part of the create process in the old cros_sdk implementation, probably will be adding it to cork at some point too. If it is a concern maybe a `--sdk-version=check` or `git` or similar keyword could be used to always perform the git clone.

Basically my thought is to use locally cached stuff when reasonable by default, that way there are fewer things to tweak or fix when I find myself working remotely with limited or no internet access and still want my system to behave in a sane way.